doc/chapter-firststeps.asciidoc

```diff
@@ -1,44 +1,38 @@
-After you've installed newsboat, you can run it for the first time by typing
-`newsboat` on your command prompt. This will bring you the following message:
+After installing newsboat, open a terminal and type `newsboat`. You'll see
+something like this:
 
 ----
-Error: no URLs configured. Please fill the file /home/ak/.newsboat/urls with RSS feed URLs or import an OPML file.
-
-newsboat 2.10
-usage: ./newsboat [-i <file>|-e] [-u <urlfile>] [-c <cachefile>] [-x <command> ...] [-h]
-    -e, --export-to-opml            export OPML feed to stdout
-    -r, --refresh-on-start          refresh feeds on start
-    -i, --import-from-opml=<file>   import OPML file
-    -u, --url-file=<urlfile>        read RSS feed URLs from <urlfile>
-    -c, --cache-file=<cachefile>    use <cachefile> as cache file
-    -C, --config-file=<configfile>  read configuration from <configfile>
-    -X, --vacuum                    compact the cache
-    -x, --execute=<command>...      execute list of commands
-    -q, --quiet                     quiet startup
-    -v, --version                   get version information
-    -l, --log-level=<loglevel>      write a log with a certain loglevel (valid values: 1 to 6)
-    -d, --log-file=<logfile>        use <logfile> as output log file
-    -E, --export-to-file=<file>     export list of read articles to <file>
-    -I, --import-from-file=<file>   import list of read articles from <file>
-    -h, --help                      this help
+Starting newsboat 2.20...
+
+'urls' file not found. Please create it at one of the following locations:
+- /home/user/.newsboat/urls
+- /home/user/.config/newsboat/urls
+
+Newsboat will store the rest of the files accordingly.
 ----
 
-This means that newsboat can't start without any configured feeds. To add
-feeds to newsboat, you can either add URLs to the configuration file
-_$HOME/.newsboat/urls_ or you can import an OPML file by running `newsboat -i
-blogroll.opml`. To manually add URLs, open the file with your favorite text
-editor and add the URLs, one per line:
+Newsboat requires a list of feed URLs, and it supports both traditional
+"dotdirs" and the modern XDG Base Directory locations
+(<<_xdg_base_directory_support,more on XDG support later>>). The _urls_ file is
```

Comment:
Was the problem caused by the word "indicator"? I'd like to adopt your suggestion with a little edit: "The placement of the urls file *indicates* the location you want to use." It's probably just pedantry, but "determines" would imply that the file dictates something to you, whereas in fact it merely *indicates* your preference.

Reply:
I tried to figure that out, but I cannot remember exactly. My best bet, however, would be the "indicator", too.

Your version is absolutely fine with me. I'm wondering whether we should stress that **only** the _urls_ file is used to determine the directory structure to use. Of course we haven't introduced the other files yet at this point (when reading straight from the top), but a more familiar user might have _config_, _cache.db_ and others in mind.

Well, the file dictates my preference *to Newsboat*.

Reply:
Okay, I'll apply my suggestion then.

> I'm wondering whether we should stress that only the urls file is used to determine the directory structure to use.

Doesn't the error message do that already? It says "Newsboat will store the rest of the files accordingly." Or by stress you mean repeating that once again, to make sure the user doesn't miss that? I'm not sure that repetition will be very effective.

Reply:
I just meant to emphasize that it's the _urls_ file and only the _urls_ file, none of the others, which is the single source of truth regarding directory structure. But yes, I agree, in combination with the program output this is actually pretty clear. I just had the documentation in mind without taking the quoted error message into account and my fictional scenario where this would help turns out to be invalid now as I think of it.